core/ir/opnd.h

```diff
@@ -1702,7 +1702,7 @@ DR_API
 opnd_t
 opnd_create_far_base_disp_ex(reg_id_t seg, reg_id_t base_reg, reg_id_t index_reg,
                              int scale, int disp, opnd_size_t size, bool encode_zero_disp,
-                             bool force_full_disp, bool disp_short_addr);
+                             bool force_full_disp, bool disp_short_addr, bool post_index);
```

Comment:
This is a breaking interface change.  This needs to be accompanied by documentation changes announcing the break.  Alternatively, instead of changing this function, a separate opnd_set_zero_offset_pre_index() could be added, which would be better for compatibility.

Reply:
I guess opnd_set_zero_offset_post_index() since the default is pre.

Reply:
Done the change accordingly in the latest commit.